code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -337,6 +337,9 @@
 	icobase_tail = 1
 
 	inherent_verbs = list(
+		/mob/living/carbon/human/proc/succubus_drain,
+		/mob/living/carbon/human/proc/succubus_drain_finalize,
+		/mob/living/carbon/human/proc/succubus_drain_lethal,
```

Comment:
i think this is it what you mean ?

Reply:
I mean, in the code, put a comment at the top and bottom of those 3 things yeah. Saying they're YW Edits.